source/app.js

```diff
@@ -1,50 +1,69 @@
 'use strict';
 
-var AWS = require('aws-sdk');
-var ssm = new AWS.SSM({region: 'us-east-1'});
+import {SSMClient, GetParametersCommand} from "@aws-sdk/client-ssm";
+
+const ssm = new SSMClient({
+    region: 'us-east-1'
+});
+
 // Parse the environment name from the lambda function name: us-east-1.dev-app-lambda-use1
-var environmentName = process.env.AWS_LAMBDA_FUNCTION_NAME.split(".")[1].split("-")[0];
+const environmentName = process.env.AWS_LAMBDA_FUNCTION_NAME.split(".")[1].split("-")[0];
 
 // If lambda name contains a uniq-id, get it. Eg. us-east-1.dev-edc-app-lambda-use1
 if (process.env.AWS_LAMBDA_FUNCTION_NAME.split(".")[1].split("-").length == 5) {
-  var uniqId = process.env.AWS_LAMBDA_FUNCTION_NAME.split(".")[1].split("-")[1] + "-";
+    var uniqId = process.env.AWS_LAMBDA_FUNCTION_NAME.split(".")[1].split("-")[1] + "-";
```

Comment:
You can make this a bit more readable and shorter this way: 

If you are not familiar with them, I rewrote it as a JS ternary - 
`condition ? true : false` - they are handy for conditional constant assignment.

I also recommend you use strict equality `===` instead of loose equality `==` - difference being that strict equals does not account for type coercion, so it's just a but more straightforward and reliable. I try to only use loose equality if I am using the type coercion aspect explicitly - in this case you don't need it - .length will always return a number. 

```
const lambdaFuncName = process.env.AWS_LAMBDA_FUNCTION_NAME.split(".")[1].split("-");
const uniqueId = lambdaFuncName.length === 5 ? lambdaFuncName : "" 
```

Reply:
Thank you @annaleeswitek!